static/app/views/dashboards/widgetBuilder/buildSteps/filterResultsStep/index.tsx

```diff
@@ -189,6 +192,23 @@ export function FilterResultsStep({
   );
 }
 
+function OnDemandWarningIcon() {
+  return (
+    <Tooltip
+      title={
+        <React.Fragment>
+          {t(
+            'We don’t routinely collect metrics from this property. However, we’ll do so '
+          )}
+          <b> {t('once this widget has been saved.')}</b>
```

Comment:
Let's use `tct` for it because this might not work for some languages. Example usage:
https://github.com/getsentry/sentry/blob/de55bc8323a4906e11856ecef01e0d33fdfe1c35/static/app/views/monitors/components/monitorForm.tsx#L257

Reply:
Works great! thx